packages/l10n/src/create-l10n-injector/create-l10n-injector.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { reportErrorToSentry } from '@commercetools-frontend/sentry';
+import { getDisplayName } from '../utils';
+
+type InjectorOptions<T> = {
+  displayName: string;
+  propKey: string;
+  propLoadingKey: string;
+  loadLocale: (locale: string, cb: (error?: Error, data?: T) => void) => void;
+};
+
+export default function createL10NInjector<T>({
+  displayName,
+  propKey,
+  propLoadingKey,
+  loadLocale,
+}: InjectorOptions<T>) {
+  return function createHOC<U>(mapPropsToLocale: (props: any) => string) {
```

Comment:
I would suggest giving types not names like `T` and `U`. To me it reminds me of Uni when we named our variables `var a,b,c,d`. I'd call it something like `PassedProps` and `AugmentedProps` or `InjectedProps`. Maybe just me though :)

Reply:
Yeah, I suggested that as well. For me personally is fine but it would be easier to read if we name them properly.

PS: it was me that said to name those T,U,... in the first place 😇